types/serverless-step-functions/types/state.d.ts

```diff
@@ -0,0 +1,58 @@
+import { Choice } from './choice';
+import { Fail } from './fail';
+import { Map } from './map';
+import { Parallel } from './parallel';
+import { Pass } from './pass';
+import { Succeed } from './succeed';
+import { Task } from './task';
+import { Wait } from './wait';
+
+export type JsonPrimitive = string | number | boolean | null;
+export interface JsonObject {
+    [member: string]: JsonValue;
+}
+export interface JsonArray extends Array<JsonValue> {}
+export type JsonValue = JsonPrimitive | JsonObject | JsonArray;
```

Comment:
Does this accept: 
`{ something: null }`

Reply:
Yes, `JsonPrimitive` contains `null`